components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -1224,7 +1224,7 @@
                     onSuccess: function(response) {
                         var patients = [];
                         response.responseJSON.each(function(item) {
-                            patients.push(item.replace("data.",""));
+                            patients.push(item.substring(item.lastIndexOf('.') + 1));
```

Comment:
Unnecessary. `lastIndexOf` returns -1 if `.` is not there, and then `substring` just returns the whole string.